packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -98,5 +168,163 @@ export async function forceLightningLwcPreview(sourceUri: vscode.Uri) {
 
     await commandlet.run();
     telemetryService.sendCommandEvent(logName, startTime);
+  } else if (desktopSelected) {
+    try {
+      await openBrowser(fullUrl);
+      telemetryService.sendCommandEvent(logName, startTime);
+    } catch (e) {
+      showError(e, logName, commandName);
+    }
+  }
+}
+
+async function selectPlatformAndExecute(
```

Comment:
Can we add JSDoc for this function?
Also, would `selectMobilePlatformAndExecute` be a better name? Since this is only relevant for mobile